package.json

```diff
@@ -16,5 +16,9 @@
   "scripts": {
     "dev": "gulp dev",
     "prestart": "gulp"
+  },
+  "engines": {
+    "node": ">=8.9.1",
+    "yarn": ">=1.3.2"
```

Comment:
This is the yarn version I generated the lock file with. Though, the latest yarn version is 1.6.0.